packages/hash/api/src/graphql/resolvers/knowledge/entity/entity.ts

```diff
@@ -252,7 +252,7 @@ export const updateEntity: ResolverFn<
 ) => {
   // The user needs to be signed up if they aren't updating their own user entity
   if (
-    entityId !== userModel.getEntityUuid() &&
+    entityId !== userModel.getBaseId() &&
```

Comment:
For anyone viewing along, this fixes an issue with non-signed up users not being able to sign up properly